### PR TITLE
Update metrics and config in documentation

### PR DESCRIPTION
**Pull request by LuQQiu**

### CODE REVIEW

File: docs/_data/table/en/common-configuration.yml [hunk 1]
```diff
@@ -313,9 +321,11 @@ fs.cos.secret.key:
 fs.cos.socket.timeout:
   'The timeout of COS socket.'
 fs.gcs.accessKeyId:
-  'The access key of GCS bucket.'
+  'The access key of GCS bucket.This property keyis only valid when alluxio.underfs.gcs.version=1'
+fs.gcs.credential.path:
+  'The json file path of Google application credentials. This property key is only valid when alluxio.underfs.gcs.version=2'
 fs.gcs.secretAccessKey:
-  'The secret key of GCS bucket.'
+  'The secret key of GCS bucket.This property keyis only valid when alluxio.underfs.gcs.version=1'
```
Comment (Xenorith) [inline docs/_data/table/en/common-configuration.yml:328]: no space between the two sentences? mistake in the underlying javadoc?

File: docs/_data/table/en/common-configuration.yml [hunk 2]
```diff
@@ -313,9 +321,11 @@ fs.cos.secret.key:
 fs.cos.socket.timeout:
   'The timeout of COS socket.'
 fs.gcs.accessKeyId:
-  'The access key of GCS bucket.'
+  'The access key of GCS bucket.This property keyis only valid when alluxio.underfs.gcs.version=1'
```
Comment (Xenorith) [inline docs/_data/table/en/common-configuration.yml:324]: no space between the two sentences? mistake in the underlying javadoc?

File: docs/_data/table/common-configuration.csv [hunk 3]
```diff
@@ -146,6 +147,9 @@ alluxio.zookeeper.leader.path,"/alluxio/leader"
 alluxio.zookeeper.session.timeout,"60s"
 aws.accessKeyId,""
 aws.secretKey,""
+awt.toolkit,""
```
Comment (Xenorith) [inline docs/_data/table/common-configuration.csv:150]: these 3 look a bit strange to me, do you know where they come from?
i found a `File.separator` call in TarUtils.java, but that was added a year ago
i also could not find any matching strings for `file.encoding`

**Comment by LuQQiu** [comment]
This PR is closed to wait for some existing metrics PR to go in.

File: docs/_data/table/en/common-configuration.yml (hunk 1 above)
Comment (LuQQiu) [reply docs/_data/table/en/common-configuration.yml:328]: Oh dannn... it's built on my local maven build, never mind

File: docs/_data/table/common-configuration.csv (hunk 3 above)
Comment (LuQQiu) [reply docs/_data/table/common-configuration.csv:150]: that could be some environment variables that not used by alluxio. For the second commit, i delete some of the parameters that maybe from jvm or the underlying package. I will remove these properties in future PR

**Pull request closed without merging**